routing/pathfind.go

```diff
@@ -193,16 +193,16 @@ func (r *Route) nextHopVertex(n *btcec.PublicKey) (Vertex, bool) {
 // nextHopChannel returns the uint64 channel ID of the next hop after the
 // target node. If the target node is not found in the route, then false is
 // returned.
-func (r *Route) nextHopChannel(n *btcec.PublicKey) (*Hop, bool) {
-	hop, ok := r.nextHopMap[NewVertex(n)]
+func (r *Route) nextHopChannel(n Vertex) (*Hop, bool) {
+	hop, ok := r.nextHopMap[n]
```

Comment:
Feel like we might as well remove this methods now?

Reply:
Only spot I see it used now is the tests.